Source/Common/Preferences.cpp

```diff
@@ -544,36 +553,43 @@ int Preferences::ExplorerShell()
         ".mpgx;mpegFile\r\n"
         ".mpls;MPLSFile\r\n"
         ".mpm;mpegFile\r\n"
+	".mpo;mpegFile\r\n"
         ".mpv;mpegFile\r\n"
         ".mts;mpegFile\r\n"
         ".mxf;MXFFile\r\n"
         ".oga;OGAFile\r\n"
         ".ogg;OGGFile\r\n"
         ".ogm;OGMFile\r\n"
         ".ogv;OGVFile\r\n"
+	".oma;ATRACFile\r\n"
         ".opus;OpusFile\r\n"
         ".png;PNGFile\r\n"
+	".pns;PNGFile\r\n"
         ".qcp;QCPFile\r\n"
         ".qt;QTFile\r\n"
         ".ra;RMFile\r\n"
         ".rm;RMFile\r\n"
         ".rmvb;FlashVideoFile\r\n"
         ".smv;SMVFile\r\n"
+	".spx;SpeexFile\r\n"
         ".stl;STLFile\r\n"
         ".swf;FlashFile\r\n"
         ".tak;TakVideoFile\r\n"
 	".tif;TIFFFile\r\n"
 	".tiff;TIFFFile\r\n"
+	".tmf;mpegFile\r\n"
         ".tp;TPFile\r\n"
         ".trp;TRPFile\r\n"
         ".ts;tsFile\r\n"
         ".tta;TTAFile\r\n"
+	".ty;mpegFile\r\n"
         ".vob;mpegFile\r\n"
         ".w64;W64File\r\n"
         ".wav;WAVFile\r\n"
         ".webM;WebMFile\r\n"
         ".wma;WMAFile\r\n"
         ".wmv;WMVFile\r\n"
+	".wtv;mpegFile\r\n"

```

Comment:
Done.

I didn't even know anything about this format before make format test support for an other project.
Thank to _mplayer.ru_ for providing many samples ;-)

I don't know if you are interested but it's look that PotPlayer is able to do that.
I found an code source, but i don't know if that time it was supporting it as well ?
In case you'll find some interest, i can leave the link here:
hg clone http://hg.code.sf.net/p/potplayer/code potplayer-code